### ISSUE TITLE
Add global API to open the Lightbox

### ISSUE THREAD

**Issue by arnowelzel** [arnowelzel/lightbox-photoswipe#66]
Also see https://wordpress.org/support/topic/opening-photoswipe-via-javascript-api/

There may be usecases which require to open the lightbox without clicking in image. This could be done by exposing a global function. For example:

```
window.lbwpsOpenPhotoSwipe(linkObject)

window.lbwpsOpenPhotoSwipeById(id)
```

Where `linkObject` is the image link to open and `id` would be the ID. Depending on if you have an object or just an ID you can then use the appropriate function.

Usage example:

```
<a id="image1" href="myimage.jpg" ...><img src="myimage-300x300.jpg" ...></a>

<button type="button" onClick="window.lbwpsOpenPhotoSwipeById('image1');">Open me!</button>
```

**Comment by PawelPatyk** [comment]
"window.lbwpsOpenPhotoSwipeById is not a function" :(

**Comment by arnowelzel** [comment]
Yes, because is not implemented yet. This issue only describes how it _could_ be done for future versions, not how it _is_ at the moment.

**Comment by arnowelzel** [comment]
Since you are a bit disappointed - sorry, but I maintain this project in my free time and some things don't have a high priority.

At https://wordpress.org/support/topic/opening-photoswipe-via-javascript-api/ you also find other solutions like using the `.click()` method on the image links which also opens the lightbox.

For example: on https://arnowelzel.de/en/projects/electronics/brymen-bm257s you can use the following call to open the first image there:

`document.querySelector('a[href="https://arnowelzel.de/en/wp-content/uploads/sites/2/2021/07/brymen-bm257s.jpg"]').click();`

**Comment by PawelPatyk** [comment]
I added the code below and nothing happens :(
```
$('#button-photo-swipe').on('click', function() {
    $('#image-1').click();
});
```

**Comment by PawelPatyk** [comment]
Link: <a href="img.jpg" id="image-1" data-lbwps-width="990" data-lbwps-height="643" data-lbwps-handler="1">...</a>

**Comment by arnowelzel** [comment]
Please check the result of this in the console:

`console.log(document.getElementById('image-1'))`

**Comment by PawelPatyk** [comment]
![console-log](https://user-images.githubusercontent.com/911505/149978848-fe872b59-b67c-4fe3-ac12-ed695e0b0b31.png)

**Comment by arnowelzel** [comment]
Then this should open the lightbox:

`console.log(document.getElementById('image-1')).click();`

**Comment by PawelPatyk** [comment]
`<a id="image-1" href="…/uploads/2022/01/img.jpg" data-lbwps-width="990" data-lbwps-height="643" data-lbwps-handler="1">
zwiedzanie-krakowa-stare-miasto-i-zamek-wawel:583:13
Uncaught TypeError: console.log(...) is undefined`

**Comment by PawelPatyk** [comment]
Works, thanks :)

**Comment by arnowelzel** [comment]
As I said - without `console.log`:

Then this should open the lightbox (sorry for the typo):

`document.getElementById('image-1').click();`

**Comment by arnowelzel** [comment]
Ok, then your jQuery call may not be correct. But this is something which has nothing to do with my plugin.
